### DISCUSSION TITLE
Error Creating Database Connection on Server-side Only

### DISCUSSION THREAD

**Discussion by chrstinalin** [prisma/prisma#20826] [Q&A]
### Question

Hi!

I have an API built using [this](https://www.prisma.io/blog/nestjs-prisma-rest-api-7D056s1BmOL0) Prisma blogpost.

Everything works perfectly on my local machine. I have it running on a CentOS v7.9.2009 server but, upon querying, I receive the error:

`PrismaClientInitializationError; Error creating a database connection.`

Since it works fine locally, I'm curious what the cause of this could be. Could it have something to do with how the server is set up?

I'm using a .env file for the database URL formatted as;

`DATABASE_URL="sqlserver://[server];database=[database];user=[username];password=[password];integratedSecurity=false;trustServerCertificate=true;encrypt=false;"
`

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
}

datasource db {
  provider = "sqlserver"
  url      = env("DATABASE_URL")
}
```

- OS: CentOS v7.9.2009
- Database: SQL Server
- Node.js version: v16.20.2

**Comment by ludralph** [comment]
Hello @chrstinalin 👋 

Can you connect to the hosted database using a tool like [DBeaver](https://dbeaver.io/) or [TablePlus](https://tableplus.com/)?

**Comment by chrstinalin** [comment]
@ludralph I just tried DBeaver and it looks like it connects both by this method --

![image](https://github.com/prisma/prisma/assets/44586776/61e0475d-24a2-48e3-ae4d-69484c852b3d)

And directly putting the same URL I used in the .env file, if I add jbdc to the beginning.

![image](https://github.com/prisma/prisma/assets/44586776/0b755946-2848-4682-961e-337eb742c7c8)

My project _does_ work when I run it locally and query it from there. It's only when I have it running on the server and query it that it's unable to establish a connection.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
